src/sci/impl/protocols.cljc

```diff
@@ -213,7 +217,7 @@
                       `(do
                          (clojure.core/alter-var-root
                           (var ~protocol-name) update :satisfies (fnil conj #{})
-                          (symbol (str ~type)))
+                          (symbol (~'type->str ~type)))
```

Comment:
because this var doesn't exist in clojure, syntax quote appends the `sci.impl.protocols` ns to it by default, which breaks (since we're exposing this in `clojure.core`). I work around this by unquoting and using a single quote to emit just the bare symbol.

Reply:
I suppose I can fully qualify this, actually...